packages/core/src/components/tag/_common.scss

```diff
@@ -59,84 +48,66 @@ $tag-padding-large: ($tag-height-large - $pt-icon-size-large) / 2 !default;
     border-radius: $tag-height-large / 2;
   }
 
-  #{$removable-selector} {
+  &.pt-tag-removable {
     padding-right: $tag-height-large;
   }
 }
 
-@mixin tag-minimal() {
+@mixin pt-tag-intent($background-color, $text-color: $white) {
+  background: $background-color;
+  color: $text-color;
+}
+
+@mixin pt-tag-minimal() {
   &:not([class*="pt-intent-"]) {
-    background: rgba($gray3, 0.2);
+    background-color: rgba($gray3, 0.2);
     color: $pt-text-color;
 
     .pt-dark & {
-      background: rgba($gray3, 0.2);
       color: $pt-dark-text-color;
     }
   }
+}
 
-  @each $intent, $color in $pt-intent-colors {
-    &.pt-intent-#{$intent} {
-      background: rgba($color, 0.15);
-      color: map-get($pt-intent-text-colors, $intent);
-    }
+@mixin pt-tag-minimal-intent($background-color, $text-color, $dark-text-color) {
+  background-color: rgba($background-color, 0.15);
```

Comment:
What if I want an exact color for the background (e.g. some really light color from my style guide's color palette)?